addons/test_mail/tests/test_performance.py

```diff
@@ -1875,7 +1875,7 @@ def test_message_post_loop(self):
         self.push_to_end_point_mocked.reset_mock()  # reset as executed twice
         self.flush_tracking()
 
-        with self.assertQueryCount(employee=800):  # tm: 791
```

Comment:
Could you add a comment here on diff for each query count to explain what are the extra queries?